flixel/addons/effects/IFlxEffect.hx

```diff
@@ -0,0 +1,12 @@
+package flixel.addons.effects;
+import openfl.display.BitmapData;
+import openfl.geom.Point;
+
+interface IFlxEffect
+{
+	public var active:Bool;
+	public var offset:Point;
+	public function destroy():Void;
```

Comment:
Ok, I will make all this changes today, and review everything to see what has changed and what can be improved here.